superset/assets/src/visualizations/PairedTTest/TTestTable.jsx

```diff
@@ -1,15 +1,24 @@
-import d3 from 'd3';
 import dist from 'distributions';
-
 import React from 'react';
 import { Table, Tr, Td, Thead, Th } from 'reactable';
-import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 
-import './paired_ttest.css';
+const propTypes = {
+  metric: PropTypes.string.isRequired,
+  groups: PropTypes.array.isRequired,
+  data: PropTypes.array.isRequired,
```

Comment:
still can't quite tell what group is:

`[ { group: [array of something?] values: [number, number] } ]`

```javascript
  PropTypes.arrayOf(
    PropTypes.shape({ 
      group: PropTypes.arrayOf(??), 
      values: PropTypes.arrayOf(PropTypes.number), 
    }),
  )
```